instance/taskinst.go

```diff
@@ -276,6 +276,9 @@ func (ti *TaskInst) EvalActivity() (done bool, evalErr error) {
 
 	eval := true
 
+	// Start Trace
+	ti.traceContext, _ = trace.GetTracer().StartTrace(ti.SpanConfig(), ti.flowInst.tracingCtx)
```

Comment:
can we guard against these method calls if someone isn't using tracing?  Possibly  something like:
 ```
if t := trace.GetTracer(); t != nil {
    ti.traceContext, _ = t.StartTrace(ti.SpanConfig(), ti.flowInst.tracingCtx)
}
```